include/sway/tree/view.h

```diff
@@ -317,6 +317,7 @@ void view_map(struct sway_view *view, struct wlr_surface *wlr_surface,
 void view_unmap(struct sway_view *view);
 
 void view_update_size(struct sway_view *view, int width, int height);
+void view_center(struct sway_view *view);
```

Comment:
Maybe rename to `view_center_surface`? `view_center` sounds like the function would center the view on screen.

Reply:
Agreed